code/_onclick/other_mobs.dm

```diff
@@ -28,11 +28,16 @@
 	if(SEND_SIGNAL(src, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, A, proximity) & COMPONENT_NO_ATTACK_HAND)
 		return
 	SEND_SIGNAL(src, COMSIG_HUMAN_MELEE_UNARMED_ATTACK, A, proximity)
+	var/rmb_stam_penalty = 1
+	if(istype(rmb_intent, /datum/rmb_intent/strong) || istype(rmb_intent, /datum/rmb_intent/swift))
+		rmb_stam_penalty = 1.5	//Uses a modifer instead of a flat addition, will be less no matter what. It's unarmed, so faster movement and less weight. 50% over whatever release drain is.
```

Comment:
`will be less no matter what` huh? like, less than weapons use?

Reply:
yeah that's what i meant; will fix the commented out stufff